app/service/rest.py

```diff
@@ -181,10 +185,6 @@ def update_service(service_id):
     update_dict = service_schema.load(current_data).data
     dao_update_service(update_dict)
 
-    # bridging code between frontend is deployed and data has not been migrated yet. Can only update current year
-    if 'free_sms_fragment_limit' in req_json:
-        update_free_sms_fragment_limit_data(fetched_service.id, req_json['free_sms_fragment_limit'])
-
     if service_going_live:
```

Comment:
We need this. Updating free_sms_fragment)limit still go through the service_api_client in admin.